src/middlewared/middlewared/plugins/alert.py

```diff
@@ -279,20 +279,25 @@ def __init__(self, middleware):
     @private
     async def node_map(self):
         nodes = {
-            "A": "Active Controller",
-            "B": "Standby Controller",
+            'A': 'Controller A',
+            'B': 'Controller B',
         }
-        if (
-            not await self.middleware.call('system.is_freenas') and
-            await self.middleware.call('failover.licensed') and
-            (
-                (await self.middleware.call('failover.node') == 'A' and
-                 await self.middleware.call('failover.status') == 'BACKUP') or
-                (await self.middleware.call('failover.node') == 'B' and
-                 await self.middleware.call('failover.status') == 'MASTER')
-            )
-        ):
-            nodes["A"], nodes["B"] = nodes["B"], nodes["A"]
+        if not await self.middleware.call('system.is_freenas') and await self.middleware.call('failover.licensed'):
+            node = await self.middleware.call('failover.node')
+            status = await self.middleware.call('failover.status')
+            if status == 'MASTER':
+                if node == 'A':
+                    nodes = {
+                        'A': 'Active Controller (A)',
+                        'B': 'Standby Controller (B)',
+                    }
+                else:
+                    nodes = {
+                        'A': 'Standby Controller (A)',
+                        'B': 'Active Controller (B)',
+                    }
+            else:
+                nodes[node] = f'{status.title()} Controller ({node})'
```

Comment:
That would give us "Master Controller" and "Backup Controller"?

Is that really what we want?

Reply:
@themylogin ping

Reply:
In that case seems we should just say "Controller A/B" ?

Reply:
I agree it will sound less confusing for user but I don't want to hide from us what was the failover state when this happened and make debugging harder if not impossible. This will be (should be) a rare case when user sees this and when he sees this I want to have at least some clue on what's going on.

Reply:
Fair enough